File: Selenium/Project/LMS_Activity3.java
package project;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class LMS_Activity3 {
    public static void main(String[] args) {
      
        System.setProperty("webdriver.chrome.driver", "path/to/chromedriver");

        
        WebDriver driver = new ChromeDriver();

        try {
           
            driver.get("https://alchemy.hguy.co/lms");

           
            WebElement firstInfoBoxTitle = driver.findElement(By.xpath("//div[contains(@class,'elementor-icon-box-wrapper')][1]//h3"));

          
            String infoBoxTitleText = firstInfoBoxTitle.getText();

           
            if ("Actionable Training".equals(infoBoxTitleText)) {
                System.out.println("Info box title matches successfully.");
            } else {
                System.out.println("Info box title does not match. The actual title is: " + infoBoxTitleText);
            }
        } catch (Exception e) {
            e.printStackTrace();
        } finally {
            
            driver.quit();
        }
    }
}

File: Selenium/Project/lms_Activity3.py
# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By

    # Start the Driver
with webdriver.Firefox() as driver:

    # Navigate to the URL
    driver.get("https://alchemy.hguy.co/lms/")
    driver.maximize_window()

    # Get the heading of the webpage.
    firstInfo_Title = driver.find_element(By.XPATH, "//div[@id='uagb-infobox-f08ebab0-fbf1-40ec-9b2a-c9feeb3d4810']/div[1]/div[1]/div[1]/div[2]/h3[1]")

    # Verify the heading of the webpage.
    if(firstInfo_Title.text =="Actionable Training"):
        print ("The title of the first info box is : ",firstInfo_Title.text)
    else:
        print ("Wrong Website ") 

    #close the browser
    driver.quit()